tests/runner.py

```diff
@@ -54,7 +58,11 @@ def nonzero_return_code_for_failures(self):
             result = run("false", warn=True)
             eq_(result.exited, 1)
             result = run("goobypls", warn=True, hide='both')
-            eq_(result.exited, 127)
+            if WINDOWS:
+                # Windows doesn't give a 127 error for "unknown command"
+                assert result.exited != 0
```

Comment:
I think it's just 1, but I couldn't be sure it's the same everywhere (Windows doesn't have a tradition of well-defined "system-level" exit codes like this). So I just left it as non-zero as a safe option. I'm happy enough to say "equals 1" and fix it later if that's too specific.